src/components/PeopleFilters.tsx

```diff
@@ -1,21 +1,68 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+import { CenturiesOptions, SexOptions } from '../utils/data';
+
+type Props = {
+  sex: SexOptions;
+  centuries: CenturiesOptions[];
+  query: string;
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void,
+};
+
+export const PeopleFilters: React.FC<Props> = ({
+  sex,
+  centuries,
+  query,
+  handleQueryChange,
+}) => {
+  const centuryParams = (century: CenturiesOptions) => {
+    return centuries.includes(century)
+      ? centuries.filter(num => century !== num)
+      : [...centuries, century];
+  };
+
+  const resetParams = {
```

Comment:
The resetParams object should not contain null values. Instead, you should use appropriate initial values for centuries, sex, and query. For example, an empty array for centuries, an empty string for sex and query.

Reply:
![image](https://github.com/mate-academy/react_people-table-advanced/assets/90213755/bc31001e-3103-43b1-9eef-14f28d9efc6f)
If i change null to initial values after Reset i will get 
![image](https://github.com/mate-academy/react_people-table-advanced/assets/90213755/262160c8-7306-4a9f-865d-b455ea2fd6c4)
I think its not correct )